includes/Providers/ServiceProvider.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace StorePulse\StoreGrowth\Providers;
+
+use StorePulse\StoreGrowth\DependencyManagement\BootableServiceProvider;
+use StorePulse\StoreGrowth\REST\ProductController;
+
+class ServiceProvider extends BootableServiceProvider {
```

Comment:
- Move the provider to `StorePulse\StoreGrowth\DependencyManagement\Providers` namespace
- Rename to `RestServiceProvider`
- Register to https://github.com/getdokan/storegrowth-sales-booster/blob/eddd2ffa6f973855abd6d4d90fbdb89db3c1915b/includes/DependencyManagement/Providers/ServiceProvider.php#L30